azurerm/internal/services/containers/container_registry_resource.go

```diff
@@ -170,6 +170,64 @@ func resourceArmContainerRegistry() *schema.Resource {
 				},
 			},
 
+			"policies": {
+				Type:       schema.TypeList,
+				MaxItems:   1,
+				Computed:   true,
+				Optional:   true,
+				ConfigMode: schema.SchemaConfigModeAttr,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"retention_policy": {
+							Type:       schema.TypeList,
+							MaxItems:   1,
+							Computed:   true,
+							Optional:   true,
+							ConfigMode: schema.SchemaConfigModeAttr,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"days": {
+										Type:     schema.TypeInt,
+										Optional: true,
+									},
+
+									"status": {
+										Type:     schema.TypeString,
+										Default:  string(containerregistry.Disabled),
+										Optional: true,
+										ValidateFunc: validation.StringInSlice([]string{
+											string(containerregistry.Disabled),
+											string(containerregistry.Enabled),
+										}, false),
+									},
+								},
+							},
+						},
+
+						"trust_policy": {
+							Type:       schema.TypeList,
+							MaxItems:   1,
+							Computed:   true,
+							Optional:   true,
+							ConfigMode: schema.SchemaConfigModeAttr,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"status": {
```

Comment:
could we switch this to a boolean?
```suggestion
									"enabled": {
```